docs/docs/guides/customize-identity-model.mdx

```diff
@@ -0,0 +1,293 @@
+
+- When finished editing name your custom Identity Model. Input the name in the
+  textbox between the checkbox and the schema.
+
+- Press `Enter` or hit the `Update` button to save it, once you have made your
+  changes. Currently, it is not possible to edit an existing Identity Model on the same revision. The identity model can still be reused as a template for a following revision.
+
+You can specify a title for your custom Identity Model. The default is
+`"title": "Person",` but this can be changed to something that describes this
+Identity Model more accurately, e.g.:
+
+```
+"title": "Customer",
+"title": "Administrator",
+"title": "Foo",
+etc...
+```
+
+## Customize Fields
+
+An Identity Models traits are specified under
+
+```json
+"traits": {
+"type": "object",
+"properties": {
+```
+
+Each trait translates into a field on the user-facing frontend. For example the
+"email & password" preset defines two traits - email and password:
+
+```json
+// email & password identity model
+    "traits": {
+      "type": "object",
+      "properties": {
+        "email": {
+          "type": "string",
+          "format": "email",
+          "title": "E-Mail",
+          "ory.sh/kratos": {
+            "credentials": {
+              "password": {
+                "identifier": true
+              }
+            },
+            "recovery": {
+              "via": "email"
+            },
+            "verification": {
+              "via": "email"
+            }
+          }
+        }
+      },
+      "required": [
+        "email"
+      ],
+      "additionalProperties": false
+    }
+```
+
+This Identity Model translates into the following sign-up screen:
+![Email Password Identity Model](../images/identity-model/email-password.png)
+
+The part highlighted below defines the identities `email` for the
+email+password flow in the Ory Cloud. It also includes a method for
+[`recovery`](https://www.ory.sh/kratos/docs/self-service/flows/account-recovery#recovery-link-method)
+as well as
+[`verification`](https://www.ory.sh/kratos/docs/self-service/flows/verify-email-account-activation/#verification-link-method).
+Currently, only the `email` method is available (recovery/verification via a
+link sent in an email).
+
+```diff
+    "traits": {
+      "type": "object",
+      "properties": {
++       "email": {
++         "type": "string",
++         "format": "email",
++         "title": "E-Mail",
++         "ory.sh/kratos": {
++           "credentials": {
++             "password": {
++               "identifier": true
++             }
++           },
+            "recovery": {
+              "via": "email"
+            },
+            "verification": {
+              "via": "email"
+            }
+          }
+        }
+      },
+      "required": [
+        "username"
+      ],
+      "additionalProperties": false
+    }
+```
+
+To add traits to the Identity Model, add them inside the `traits "properties"`
+
+```json
```

Comment:
Use json5 if you include comments

Reply:
```suggestion
```json5
```
Do you mean this? 
It loses the color highlights if I use `json5` here. I think using `json` here, already renders it correctly.

with json5
<img width="968" alt="2" src="https://user-images.githubusercontent.com/55394283/128321235-c5018d71-948b-4607-914d-ad2805e41558.png">

with json
<img width="968" alt="1" src="https://user-images.githubusercontent.com/55394283/128321215-dfb603c8-010d-4ff9-83ff-a79267cd4c73.png">

Reply:
I think we just need to add json5 as an alias in the docs config. It would fix the markdown preview in github and potentially any CI we introduce in the future to validate markdown

Reply:
I changed it now in https://github.com/ory/docs/pull/453/commits/1ecf575728dede34d18a9ff64db96c5ee6194aaf to not require comments anymore. 
I think it is better this way. But we should still add json5 as alias. 
If you can give me some pointers, I could not figure out how to add it.